### PR TITLE
Update fileinfo with API v.0.7 changes

### PR DESCRIPTION
**Pull request by ivanredbread**
## Description

- add `appdata`
- move blocks with media info inside `content_info`

## Checklist

- [x] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

**Review by rsedykh — APPROVED** [review]

File: pyuploadcare/api/entities.py [hunk 1]
```diff
@@ -80,15 +81,90 @@ class VideoInfo(Entity):
...
+    aws_rekognition_detect_labels: Optional[
+        AWSRecognitionDetectLabelsApplicationData
+    ]
+    remove_bg: Optional[RemoveBackgroundApplicationData]
+    uc_clamav_virus_scan: Optional[UCClamAVApplicationData]
 
 
 class FileInfo(UUIDEntity):
+    """
+    video_info, image_info and rekognition_info were suppressed in API v0.7.0
```
Comment (dmitry-mukhin) [inline pyuploadcare/api/entities.py:158]:
```suggestion
    video_info, image_info and rekognition_info were removed in API v0.7.0
```

**Review by dmitry-mukhin — CHANGES_REQUESTED** [review]

File: pyuploadcare/api/entities.py (hunk 1 above)
Comment (ivanredbread) [reply pyuploadcare/api/entities.py:158]: I agree that it is rather confusing usage of `suppressed`.
There is another example of a better word for substitution as Roman suggested in [PR216](https://github.com/uploadcare/pyuploadcare/commit/e36751b6dd2878d7d4afa1d19c52ccecd46bf5ed)
(so it should be resolved after final merging into release epic branch in [PR219](https://github.com/uploadcare/pyuploadcare/pull/219))

Comment (dmitry-mukhin) [reply pyuploadcare/api/entities.py:158]: bump

File: tests/integration/test_api_resources.py [hunk 2]
```diff
@@ -24,7 +25,7 @@
 
 
 @pytest.fixture
-def group(uploadcare) -> FileGroup:
+def group(uploadcare) -> Iterator[FileGroup]:
```
Comment (dmitry-mukhin) [inline tests/integration/test_api_resources.py:28]:
```suggestion
def groups(uploadcare) -> Iterator[FileGroup]:
```

**Review by dmitry-mukhin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**